api/CSSMatrix.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api": {
+    "CSSCounterStyleRule": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSMatrix",
+        "support": {
+          "webview_android": {
+            "version_added": "45"
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": "45"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": "10",
+            "notes": "Internet Explorer implements this API as MSCSSMatrix. In version 11 the alias WebKitCSSMatrix was added."
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": true,
+            "notes": "WebKit implements this API as WebKitCSSMatrix."
+          },
+          "safari_ios": {
+            "version_added": true,
+            "notes": "WebKit implements this API as WebKitCSSMatrix."
```

Comment:
Like before, this note could be replaced with `alternative_name` set to "WebKitCSSMatrix"